src/app/rpmostree-builtin-status.c

```diff
@@ -563,6 +563,12 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
           print_kv ("Unlocked", max_key_len, unlocked);
           g_print ("%s%s", bold_suffix, red_suffix);
         }
+      const char *end_of_life_string = NULL;
+      /* look for endoflife attribute in the deployment */
+      g_variant_dict_lookup (dict, "endoflife", "&s", &end_of_life_string);
+
+      if (end_of_life_string)
+       g_print ("\nEndOfLife: %s", end_of_life_string);
```

Comment:
there is actually no reason, I just did not think about that :p (may consider more carefully to follow the style of the file next time :p). I also agree it might be better to add the red & bold for the highlights, and use print_kv as other key/value pairs. :)